aiida/orm/querybuilder.py

```diff
@@ -1773,11 +1746,25 @@ def queryhelp(self):
         The queryhelp can be used to create a copy of the QueryBuilder instance like so::
 
             qb = QueryBuilder(limit=3).append(StructureData, project='id').order_by({StructureData:'id'})
-            qb2=QueryBuilder(**qb.queryhelp)
+            qb2 = QueryBuilder(**qb.queryhelp)
 
-        :return: a queryhelp dictionary
+        This is a json-comptible dictionary.
```

Comment:
Ah, so is the dictionary json-compatible by default, despite we are not using the corresponding method?
@lekah do you want to comment?

Reply:
Since the property `queryhelp` was simply a proxy for `get_json_compatible_queryhelp` before, I guess so?

Reply:
> (..) despite us not using the corresponding method?

I am not sure, but I think you mixed the methods `get_json_compatible_queryhelp` and `_get_json_compatible`? The latter is completely unused in the code base, while the first generates the current `queryhelp` dictionary - i.e., the `queryhelp` property was a proxy for the first method.

Reply:
If I understood correctly, this is just a renaming: the instance method get_json_compatible_queryhelp is moved to the attribute .queryhelp, but it returns the same thing. However, whether the dictionary can be stored as JSON is indeed not being checked, so I can't answer whether it's the default (it was back in the day). I feel that this is outside the scope of this PR, though. I'd suggest creating a new issue for adding tests whether the queryhelp obtained is JSON compatible, and put in proper tests.

Reply:
So I should remove the line specifying that `queryhelp` returns a json-compatible dictionary at the moment?

Reply:
It has been removed.